doc/articles/platform-specific-xaml.md

```diff
@@ -133,3 +133,10 @@ More visually, platform support for the pre-defined prefixes is shown in the bel
 Where:
  * 'Win' represents Windows, and
  * 'Droid' represents Android
+
+### Cross target library
+Since version 3.0, the Uno platform prefixes have changed for cross targeted libraries. These libraries need to be compiled for every possible platform. For most platform, that does not change anything, but with the addition of Skia, the Uno platform cannot distinguish Skia and Wasm. 
```

Comment:
```suggestion
For Uno 3.0 and above, Xaml prefixes behave differently in class libraries than when used directly in application code. Specifically, it isn't possible to distinguish Skia and Wasm in a library, since both platforms use the .NET Standard 2.0 target. The `wasm` and `skia` prefixes will always evaluate to false inside of a library.
```